assets/src/components/parts/janus-input-number/InputNumber.tsx

```diff
@@ -275,7 +275,7 @@ const InputNumber: React.FC<PartComponentProps<InputNumberModel>> = ({
       {showLabel && (
         <React.Fragment>
           <label htmlFor={`${id}-number-input`} className="inputNumberLabel">
-            {label.length > 0 ? label : ''}
+            {label?.length > 0 ? label : ''}
```

Comment:
When the user clears the label of the number input, the label's `value `is set to `undefined`. As a result, attempting to execute `label.length` throws an error because label is `undefined`, and we're trying to get its length.